lib/Partitioner/Partitioner.cpp

```diff
@@ -1056,14 +1054,14 @@ llvm::Error Partitioner::loadBalancedPartitioning(Function *F,
   // Initialize runtimes and memory availability per device
   std::vector<float> deviceTime(numDevices, 0);
   std::vector<size_t> memoryAvailable(numDevices, availableMemory);
-  std::vector<NodesSetTy> nodesInPartitions(numDevices);
+  std::vector<NodesSet> nodesInPartitions(numDevices);
   std::vector<GraphMemInfo> graphMem(numDevices, GraphMemInfo{});
   std::vector<Function *> partitions(numDevices);
 
   // Compute total roofline time
   float totalRooflineTime = 0;
   for (auto &n : F->getNodes()) {
-    totalRooflineTime += getComputeTime()[&n];
+    totalRooflineTime += getComputeTime(&n);
```

Comment:
It seems that there is a case where Node `n` is not present in the compute map and now it's crashing vs creating element in a map with default ctor and returning result here.
Wondering if this now exposes some hidden bug.

Reply:
Thanks for raising this.  I did actually intend it, since I don't think it's desirable behavior to get `0` for looking up a missing node.  But thinking a bit more, I'm not actually certain what's desirable here.  I'm kind of leaning towards assert+default value, so it won't kill a release build, but would fail during development?

Reply:
@rdzhabarov  - did you encounter a failure case here where the crash happens?

Reply:
@nrsatish not under normal circumstances, but it's quite possible and would be hard crash.